launch.go

```diff
@@ -170,11 +170,11 @@ func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 	AccountServiceLogOutURL := getAccountServiceURL(r)
 	urlValues := r.URL.Query()
 	schemaURL := urlValues.Get("url")
+	delete(urlValues, "url")
```

Comment:
How come we're deleting this?

Reply:
`schema_url` in top level is the same value so when we do not have this in, the `url` value is passed to auth.go and is then added as survey metadata leading to urls appearing in both top level and survey metadata, we only want one and it has to be in top level according the ONS schema definitions

Reply:
<img width="1297" alt="Screenshot 2024-02-02 at 14 23 12" src="https://github.com/ONSdigital/eq-questionnaire-launcher/assets/144226112/a1017f1c-82a1-4912-abb1-85d125991c39">
This is what the token would appear as without that line in place.

Reply:
I guess it shouldn't be added in the first place rather than deleting it though?

Reply:
It is needed when `GenerateTokenFromDefaultsV2` is called in `launch.go` because it is a parameter. 

I did notice we use a separate method called `getSchemaClaims` which is used to add in `schema_url` in to top level metadata in `GenerateTokenFromDefaultsV2` and is defined in `auth.go`. 

One alternative I thought of was rather than deleting it, it's possible to add an `if` statement in `getSurveyMetadataFromClaims` to ignore a url value. Another thought was to rename the variable from `url` to `schema_url` within `urlValues` and then stop the use of `getSchemaClaims` but I think this wouldn't work for cir as it needs `getSchemaClaims` for it to be in the token. I believe it would have to be deleted/ignored to prevent duplication in the token.

Reply:
I see this is only for quick launch, so I think maybe `url` we should be made consistent with the actual `schema_url` field in launcher (unless there is a reason why these are different 🤔)?

Reply:
As discussed, I changed the query to `schemaURL := urlValues.Get("schema_url")` and updated the README 👍